### PR TITLE
RCAL-822 Improve Schema Info Test

### PR DESCRIPTION
**Pull request by PaulHuwe**
Resolves [RCAL-822](https://jira.stsci.edu/browse/RCAL-822)

Closes #

This PR replaces the previous test for schema_info with something more robust.

**Checklist**
- [x] Added entry in `CHANGES.rst` under the corresponding subsection
- [x] updated relevant tests
- [ ] updated relevant documentation
- [x] Passed romancal regression testing on Jenkins / PLWishMaster. Link: https://plwishmaster.stsci.edu:8081/job/RT/job/Roman-Developers-Pull-Requests/736/

### CODE REVIEW

**Comment by schlafly** [comment]
Do you understand the test failure?

**Comment by PaulHuwe** [comment]
> Do you understand the test failure?

Yeah - it is because I didn't repoint to my RAD branch. Once RAD is released, I will re-run the tests.

File: CHANGES.rst [hunk 1]
```diff
@@ -3,6 +3,8 @@
 
 - Adds test to ensure that the base ``common`` keyword groups exist within the ``schema.info`` tree. [#338]
 
+- Pleaced the previous test for ``schema_info`` with something more robust. [#344]
+
```
Comment (ddavis-stsci) [inline CHANGES.rst:7]: Pleaced? maybe Replaced?

**Review by ddavis-stsci — APPROVED** [review]
Look Good

File: CHANGES.rst [hunk 2]
```diff
@@ -3,6 +3,8 @@
 
 - Adds test to ensure that the base ``common`` keyword groups exist within the ``schema.info`` tree. [#338]
 
+- Pleaced the previous test for ``schema_info`` with something more robust. [#344]
```
Comment (PaulHuwe) [inline CHANGES.rst:6]:
```suggestion
- Replaced the previous test for ``schema_info`` with something more robust. [#344]
```

**Comment by PaulHuwe** [comment]
Added regtest to ensure it would not affect the repository.

**Pull request merged**